.gitignore

```diff
@@ -1,4 +1,7 @@
+python27-alt.spec
+#python27-alt.spec-me
```

Comment:
Can you remove these from this file?  We should only include files in .gitignore that should in fact be there, but should not be kept under source control.